### PR TITLE
Add missing `gcloud auth login` step to "Deploy using CLI" doc

### PR DESCRIPTION
**Pull request by Bobgy**
/assign @sarahmaddox 

@rmgogogo found this step is also required for following actions.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1484)

### CODE REVIEW

File: content/docs/gke/deploy/deploy-cli.md [hunk 1]
```diff
@@ -51,6 +51,12 @@ some handy environment variables:
     tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz
     ```
 
+1. Login. You only need to run this command once:
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/deploy-cli.md]: s/Login/Log in

When using "log in" as a verb like the above, it's two words. When using "login" as a noun or adjective, it's one word. For example:
* "I want to log in."
* "I can't find the login button."

The pattern is the same for other words/phrases. Like:

* "I want to sign up but the signup process is ridiculously long."
* "I did back up my data but the backup failed."

I wrote a blog post about it once: https://ffeathers.wordpress.com/2018/05/20/login-or-log-in-sign-up-or-signup-how-to-tell-when-to-use-one-word-when-two/

:)

Comment (Bobgy) [reply content/docs/gke/deploy/deploy-cli.md]: TIL, thanks for the detailed explanation

Comment (sarahmaddox) [reply content/docs/gke/deploy/deploy-cli.md]: :)

**Pull request merged**